### PR TITLE
Feature/consumer api hands on examples

### LINKED ISSUE #95: "Hands On" Beispielimplementierung für die Consumer API
#### Describe your issue

Analog zur "Hands On" Dokumentation zum BPMN Studio, sollte es auch eine Schritt-für-Schritt Anleitung für die TS Implementierung der Consumer API geben.

Es sollte ebenfalls einmal geprüft werden, ob sich das Handbuch so umformulieren lässt., dass man möglichst jeden Bezug zu IoC meidet,
da die Anwendung, in welcher der Consumer API Client zum tragen kommt ggf. garnicht über IoC verfügt, bzw. verfügen soll.

#### Issue checklist

Please check the boxes in this list after submitting your Issue:

- [x] I've checked if this issue already exists
- [x] I've included all the information that i think is relevant
- [x] I've added logs and/or screenshots (if applicable)
- [x] I've mentioned PRs and issues that relate to this one

### PR DESCRIPTION
**Pull request by ElRaptorus**
Closes #95 

## What did you change?

- Restructure the consumer api doc files into subfolders
- Adds references to the hands-on docs for the consumer api.

## PR-Checklist

Please check the boxes in this list after submitting your PR:

- [x] You can merge this PR **right now** (if not, please prefix the title with "WIP: ")
- [x] I've tested **all** changes included in this PR.
- [x] I've also reviewed this PR myself before submitting (e.g. for scrambled letters, typos, etc.)
- [x] I've merged the `develop` branch into my branch before finishing this PR.
- [x] I've **not added any other changes** than the ones described above.
- [x] I've mentioned all **PRs, which relate to this one**

### CODE REVIEW

**Comment by ElRaptorus** [comment]
https://github.com/process-engine-pro/organization/issues/140

File: processengine/typescript/consumer_api/hands_on/README.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Hands-On Beispiele
+
+Auf [Github](https://github.com/process-engine/consumer_api_meta) finden sich mehrere Beispielprojekte, die veranschaulichen, wie man
+die ConsumerAPI auf in verschiedenen Anwendungsszenarien implementiert.
```
Comment (5ebastianMeier) [inline processengine/typescript/consumer_api/hands_on/README.md]: * -auf

Comment (5ebastianMeier) [reply processengine/typescript/consumer_api/hands_on/README.md]: * statt `implementiert` besser `verwendet`

**Comment by ElRaptorus** [comment]
@5ebastianMeier done

**Pull request merged**